### PR TITLE
Modifying the theming.md file for better understanding

### PR DESCRIPTION
**Pull request by Ashray123**
Split the css and JSON based themes to provide clear understanding of the themes.

Fixes: #16096

Signed-off-by: ASHRAY <jha.ashray12@gmail.com>

### CODE REVIEW

File: docs/theming.md [hunk 1]
```diff
@@ -24,8 +24,8 @@ To define a theme for Element:
 In future, the assets for a theme will probably be gathered together into a
 single directory tree.
 
-Custom Themes
-=============
+ JSON Based Themes (Labs Feature):
+==================================
 
 Themes derived from the built in themes may also be defined in settings.
```
Comment (aaronraimist) [inline docs/theming.md:30]:
```suggestion
When [`feature_custom_themes`](https://github.com/vector-im/element-web/blob/master/docs/labs.md#custom-themes-feature_custom_themes) is enabled, JSON based themes derived from the built in themes may also be defined in settings.
```

**Review by aaronraimist — COMMENTED** [review]
Looks good to me. Thanks!

I'm not on the team though, you'll still need to wait for review from them.

File: docs/theming.md [hunk 2]
```diff
@@ -24,10 +24,10 @@ To define a theme for Element:
 In future, the assets for a theme will probably be gathered together into a
 single directory tree.
 
-Custom Themes
-=============
+ JSON Based Themes (Labs Feature):
+==================================
 
-Themes derived from the built in themes may also be defined in settings.
+When [`feature_custom_themes`](https://github.com/vector-im/element-web/blob/master/docs/labs.md#custom-themes-feature_custom_themes) is enabled, JSON based themes derived from the built in themes may also be defined in settings.
```
Comment (t3chguy) [inline docs/theming.md:30]: This isn't true though, you don't need the Lab enabled to have a custom theme in config.json - the labs flag lets the user add such a custom theme by a JSON URL.

Comment (Ashray123) [reply docs/theming.md:30]: @t3chguy thanks for reviewing! what further modifications do you suggest in order to make this pr a valid one?.

File: docs/theming.md [hunk 3]
```diff
@@ -24,10 +24,10 @@ To define a theme for Element:
 In future, the assets for a theme will probably be gathered together into a
 single directory tree.
 
-Custom Themes
-=============
+ JSON Based Themes (Labs Feature):
+==================================
```
Comment (aaronraimist) [inline docs/theming.md:28]:
```suggestion
JSON Based Themes:
==================
```

File: docs/theming.md (hunk 2 above)
Comment (aaronraimist) [reply docs/theming.md:30]: GitHub won't let me suggest on this line but I guess you could change it to something like

```
JSON themes derived from the built in themes may be defined in the `config.json` file. When [`feature_custom_themes`](https://github.com/vector-im/element-web/blob/master/docs/labs.md#custom-themes-feature_custom_themes) is enabled, users can add new JSON themes in the Appearance section of Settings.
```

**Comment by Johennes** [comment]
Closing since this hasn't seen any actions in two years and theming is going to change with Compound.

**Pull request closed without merging**